robot/src/main/kotlin/frc/robot/commands/Climb.kt

```diff
@@ -102,7 +103,11 @@ fun ClimbToNext(climber: WinchSubsystem, solenoid: SolenoidSubsystem): Command {
 // Automatic climbing sequence.
 // This should be started with the climber raised over the bar.
 fun AutoClimb(climber: WinchSubsystem, solenoid: SolenoidSubsystem): Command {
-    return SequentialCommandGroup(
+    if (!climber.atLower()){
+        SmartDashboard.putString("Elevator Warning", "Elevator is not lowered.")
+        return SequentialCommandGroup(WaitCommand(0.5))
+    } else{
```

Comment:
Because `AutoClimb` is intended (and will only really work) if it is started with the climber raised over the mid bar, this lockout will be triggered whenever `AutoClimb` is run.

I think the safety concern that Pascal was referring to was that the `AutoClimb` would run without ever having been zeroed. If the robot was powered on with the elevator started above the physical zero position, this would cause the elevator to run into the upper limit switch at high velocity, which could be mechanically problematic.

I think the appropriate solution would be track (probably in `WinchSubsystem`) whether the lower limit switch has ever been hit and display a warning based on that. I would also urge this to be a warning (such as controller1 rumble) instead of a total lockout.